src/main/java/org/gnieh/logback/config/ConfigConfigurator.java

```diff
@@ -248,4 +277,38 @@ private void configureLogger(LoggerContext loggerContext, Map<String, Appender<I
 
 	}
 
+	@SuppressWarnings({ "unchecked" })
+	private Object configureObject(LoggerContext loggerContext, Map<String, Object> params) throws ReflectiveOperationException {
+		Object object = null;
+		if (params.containsKey("class")) {
+			object = Class.forName((String) params.get("class")).newInstance();
+		} else {
+			throw new RuntimeException("No class in " + params.toString());
```

Comment:
What if the property is of the object configured by this type `Map<String, Object>` or `Config`? This should not result in an exception. I may want to use Config properties in my logger or appender.